backend/api/types.py

```diff
@@ -0,0 +1,6 @@
+import strawberry
+
+
+@strawberry.type
+class OperationResult:
```

Comment:
The idea was to have a simple "type" when the mutation doesn't return any information more than "operation completed with success/received"